EasyPost.Net/ApiKey.cs

```diff
@@ -18,16 +16,11 @@ public class ApiKey : Resource
         /// <exception cref="ResourceAlreadyCreated">Address already has an id.</exception>
         public static List<ApiKey> All()
         {
-            Request request = new Request("api_keys");
-            Dictionary<string, object> response = request.Execute<Dictionary<string, object>>();
-            List<object> keys = (List<object>)response["keys"];
-            foreach (Dictionary<string, object> key in keys)
+            Request request = new Request("api_keys")
             {
-                key["created_at"] = DateTime.ParseExact((string)key["created_at"], "yyyy-MM-ddTHH:mm:ssZ",
-                    CultureInfo.InvariantCulture);
-            }
-
-            return keys.Select(key => LoadFromDictionary<ApiKey>((Dictionary<string, object>)key)).ToList();
```

Comment:
Why was all this removed? (why was it all here initially?)

Reply:
For some reason, we were bypassing the JSON object deserialization downstream, and parsing it here because it (supposedly) needed to modify the `created_at` value for each entry. Turns out it doesn't (at least, not anymore), so we can let it use the deserializer downstream like everything else, and remove all this clutter here.

Reply:
I'll trust your judgement, will be good to unit test this in the next PR to ensure.